rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -567,6 +567,55 @@ rocalResize(
     return output;
 }
 
+RocalImage  ROCAL_API_CALL
+rocalResizeMirrorNormalize(
+            RocalContext p_context,
+            RocalImage p_input,
+            unsigned dest_width,
+            unsigned dest_height,
+            std::vector<float> &mean,
+            std::vector<float> &std_dev,
+            bool is_output,
+            RocalIntParam p_mirror)
+{
+    if(!p_context || !p_input || dest_width == 0 || dest_height == 0 )
+        THROW("Null values passed as input")
+    Image* output = nullptr;
+    auto context = static_cast<Context*>(p_context);
+    auto input = static_cast<Image*>(p_input);
+    auto mirror = static_cast<IntParam *>(p_mirror);
+    for(unsigned i = 0; i < mean.size(); i++) {
+        mean[i] = 0;
+        std_dev[i] = 1;
```

Comment:
Why are we hardcoding mean and std_dev value?

Reply:
In the current image pipeline, the mean and std values were not being passed to the RMN calls and instead 0 and 1 is being passed. In case of CMN, we directly [pass](https://github.com/SundarRajan28/MIVisionX/blob/RMN_dev/rocAL/rocAL/source/api/rocal_api_augmentation.cpp#L1716) 0 and 1 to the CMN node init. But RMN Tensor needs mean and std as vectors so we are storing 0 and 1 in the vectors being passed to the RMN node